### PR TITLE
Post-2.1.0-release fixes (2.1.1)

### PR DESCRIPTION
**Pull request by joneit**

### CODE REVIEW

**Comment by joneit** [comment]
Hey @Dwaynekj - I added a commit to this to revert the breaking change so we can publish to npm without further concern.

File: src/lib/DataSourceOrigin.js [hunk 1]
```diff
@@ -79,7 +82,7 @@ var DataSourceOrigin = DataSourceBase.extend('DataSourceOrigin',  {
                 schema[i] = { name: fields[i] };
             }
         }
-
+        schema = setInternalColSchema.call(this, schema);
```
Comment (Dwaynekj) [inline src/lib/DataSourceOrigin.js]: What does this accomplish?

Comment (joneit) [reply src/lib/DataSourceOrigin.js]: Just reverting the breaking change back to [__your__ code](https://github.com/fin-hypergrid/core/commit/c540b45ace615d757147861a48c26460493455b1#diff-4a3b59fc30ab47d3eee9791d7bc35230R335)!

Comment (Dwaynekj) [reply src/lib/DataSourceOrigin.js]: ok

**Comment by joneit** [comment]
@Dwaynekj Pushed another commit to resolve an issue I just fell across.

**Pull request merged**